felix/fv/bpf_test.go

```diff
@@ -631,6 +631,18 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 				})
 			})
 
+			Describe("with DefaultEndpointToHostAction=RETURN", func() {
```

Comment:
This is only right if the iptables INPUT chain policy is ACCEPT.  To be sure it's doing the right thing, we need to test both cases of the iptables policy.

Reply:
yes, that is another step in fixing a broader issue